lib/zuora/rest.rb

```diff
@@ -3,6 +3,9 @@ module Zuora
   module Rest
     API_URL = 'https://api.zuora.com/rest/v1/'.freeze
     SANDBOX_URL = 'https://apisandbox-api.zuora.com/rest/v1/'.freeze
+    # Newest endpoints for REST API calls
+    BETA_API_URL = 'https://rest.zuora.com/v1/'.freeze
+    BETA_SANDBOX_URL = 'https://rest.apisandbox.zuora.com/v1/'.freeze
```

Comment:
I agree, the change you mentioned will require a major version bump for this project. Looks like we'll both have to continue forking or monkey-patching for the time being.